src/features/lookup_table/lookup_table_feature.py

```diff
@@ -26,3 +28,17 @@ def create_lookup(recipe_package: str, application: str, user: User = Depends(au
     This can be used for setting Ui- and StorageRecipes for specific applications.
     """
     return create_lookup_table_use_case(recipe_package, application, user)
+
+
+@router.get(
+    "/application/{application}",
+    operation_id="get_lookup",
+    response_model=Lookup,
+    responses={**responses},
+)
+@create_response(JSONResponse)
+def get_lookup(application: str, user: User = Depends(auth_w_jwt_or_pat)):
```

Comment:
not sure what value "application" should have.
name or uid?

(Also see other comment)

Reply:
There are no UUID anywhere related to lookups. Only "name" which also is used as the database key/_id

Reply:
I want to hide the "_id" logic from API users. Therefor it's only refered to as "application" in the endpoints.
But internally in the code, it's also acts as an ID.